src/Meshes.jl

```diff
@@ -25,6 +25,7 @@ include("angles.jl")
 include("geometries.jl")
 include("connectivities.jl")
 include("mesh.jl")
+include("conventions.jl")
```

Comment:
So conventions.jl could come before the mesh.jl file because it is more closely related to connectivities than it is to meshes.